Objects/longobject.c

```diff
@@ -3874,6 +4000,36 @@ fast_floor_div(PyLongObject *a, PyLongObject *b)
     return PyLong_FromLong(div);
 }
 
+#ifdef WITH_PYLONG_MODULE
+/* asymptotically faster divmod, using _pylong.py */
+static int
+pylong_int_divmod(PyLongObject *v, PyLongObject *w,
+                  PyLongObject **pdiv, PyLongObject **pmod)
+{
+    PyObject *mod = PyImport_ImportModule("_pylong");
+    if (mod == NULL) {
+        return -1;
+    }
+    PyObject *r = PyObject_CallMethod(mod, "int_divmod", "OO", v, w);
+    if (r == NULL) {
+        Py_DECREF(mod);
+        return -1;
+    }
+    assert(PyTuple_Check(r));
+    PyObject *a = PyTuple_GET_ITEM(r, 0);
+    PyObject *b = PyTuple_GET_ITEM(r, 1);
+    Py_INCREF(a);
+    Py_INCREF(b);
+    assert(PyLong_Check(a));
+    assert(PyLong_Check(b));
+    *pdiv = (PyLongObject *)a;
+    *pmod = (PyLongObject *)b;
```

Comment:
`pdiv` or `pmod` could be `NULL` at the call site in `l_divmod`.

This crashed when I ran pyperformance's bm_pidigits.

Reply:
Should be fixed.